eng/tools/internal/ci.yml

```diff
@@ -2,14 +2,14 @@
 trigger:
   paths:
     include:
-      - tools/deprecate
+      - eng/tools/internal
```

Comment:
Something funky is going on with the git mv/diff here as it's correlating the wrong ci.yml files. All the files have been moved correctly with the proper paths.